laser_encoders/README.md

```diff
@@ -0,0 +1,106 @@
+# LASER encoders
+
+LASER Language-Agnostic SEntence Representations Toolkit
+
+laser_encoders is the official Python package for the Facebook [LASER](https://github.com/facebookresearch/LASER) library. It provides a simple and convenient way to use LASER embeddings in Python. It allows you to calculate multilingual sentence embeddings using the LASER toolkit. These embeddings can be utilized for various natural language processing tasks, including document classification, bitext filtering, and mining.
+
+## Dependencies
+
+- Python >= 3.8
```

Comment:
Maybe we can just drop these and simply point users to `requirements.txt`.

Reply:
Alright, Though i feel having the major requirements right here give a better user experience

Reply:
Okay, well if so we might as well be consistent. If you'd like to keep the major requirements here, then can you add in the suggested versions for each package (not just Python + PyTorch)